slices/udev.yaml

```diff
@@ -0,0 +1,176 @@
+      /usr/lib/udev/rules.d/60-drm.rules:
+      /usr/lib/udev/rules.d/60-evdev.rules:
+      /usr/lib/udev/rules.d/60-fido-id.rules:
+      /usr/lib/udev/rules.d/60-infiniband.rules:
+      /usr/lib/udev/rules.d/60-input-id.rules:
+      /usr/lib/udev/rules.d/60-persistent-alsa.rules:
+      /usr/lib/udev/rules.d/60-persistent-input.rules:
+      /usr/lib/udev/rules.d/60-persistent-storage-mtd.rules:
+      /usr/lib/udev/rules.d/60-persistent-storage-tape.rules:
+      /usr/lib/udev/rules.d/60-persistent-storage.rules:
+      /usr/lib/udev/rules.d/60-persistent-v4l.rules:
+      /usr/lib/udev/rules.d/60-sensor.rules:
+      /usr/lib/udev/rules.d/60-serial.rules:
+      /usr/lib/udev/rules.d/61-persistent-storage-android.rules:
+      /usr/lib/udev/rules.d/64-btrfs.rules:
+      /usr/lib/udev/rules.d/70-camera.rules:
+      /usr/lib/udev/rules.d/70-joystick.rules:
+      /usr/lib/udev/rules.d/70-memory.rules: { arch: [amd64, i386, armhf, arm64] }
+      /usr/lib/udev/rules.d/70-mouse.rules:
+      /usr/lib/udev/rules.d/70-power-switch.rules:
+      /usr/lib/udev/rules.d/70-touchpad.rules:
+      /usr/lib/udev/rules.d/71-power-switch-proliant.rules:
+      /usr/lib/udev/rules.d/73-special-net-names.rules:
+      /usr/lib/udev/rules.d/75-net-description.rules:
+      /usr/lib/udev/rules.d/75-probe_mtd.rules:
+      /usr/lib/udev/rules.d/78-graphics-card.rules:
+      /usr/lib/udev/rules.d/78-sound-card.rules:
+      /usr/lib/udev/rules.d/80-debian-compat.rules:
+      /usr/lib/udev/rules.d/80-drivers.rules:
+      /usr/lib/udev/rules.d/80-net-setup-link.rules:
+      /usr/lib/udev/rules.d/81-net-dhcp.rules:
+      /usr/lib/udev/rules.d/90-iocost.rules:
+
+  systemd-udevd:
+    essential:
+      - systemd-dev_dbus-interfaces
+      - systemd_bins
+      - udev_bins
+      - udev_config
+      - udev_generator
+      - udev_rules
+      - udev_sysusers-config
+      - udev_tmpfiles-config
+    contents:
+      /usr/lib/systemd/network/73-usb-net-by-mac.link:
+      /usr/lib/systemd/network/99-default.link:
+      /usr/lib/systemd/system/sockets.target.wants/systemd-udevd-control.socket:
+      /usr/lib/systemd/system/sockets.target.wants/systemd-udevd-kernel.socket:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-hwdb-update.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-udev-trigger.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-udevd.service:
+      /usr/lib/systemd/system/systemd-hwdb-update.service:
+      /usr/lib/systemd/system/systemd-udev-settle.service:
+      /usr/lib/systemd/system/systemd-udev-trigger.service:
+      /usr/lib/systemd/system/systemd-udevd-control.socket:
+      /usr/lib/systemd/system/systemd-udevd-kernel.socket:
+      /usr/lib/systemd/system/systemd-udevd.service:
+      /usr/lib/systemd/system/systemd-udevd.service.d/syscall-architecture.conf:
+      /usr/lib/systemd/system/udev.service:
+      /usr/lib/systemd/systemd-udevd:
+
+  sysusers-config:
```

Comment:
Does this make sense without `systemd-sysusers`? Which is a dep missing from this PR